backend/conf.js

```diff
@@ -10,6 +17,6 @@ const db = mysql.createPool({
 });
 
 module.exports = {
-  backendPort,
+  CONFIG,
```

Comment:
(Hmm, I don't really like the CONFIG sub-object with one part of the variables, but it works and doesn't break anything)